cli/command/formatter/container_test.go

```diff
@@ -323,3 +324,37 @@ func TestContainerContextWriteWithNoContainers(t *testing.T) {
 		out.Reset()
 	}
 }
+
+func TestContainerContextWriteJSON(t *testing.T) {
+	containers := []types.Container{
+		{ID: "containerID1", Names: []string{"/foobar_baz"}, Image: "ubuntu"},
+		{ID: "containerID2", Names: []string{"/foobar_bar"}, Image: "ubuntu"},
+	}
+	out := bytes.NewBufferString("")
+	err := ContainerWrite(Context{Format: "{{json .}}", Output: out}, containers)
```

Comment:
A test for `{{json .ID}}` or something that uses the fields would be good to have.